src/modules/cmdpal/Core/Microsoft.CmdPal.Core.ViewModels/ListItemViewModel.cs

```diff
@@ -49,14 +51,19 @@ public override void InitializeProperties()
         }
 
         UpdateTags(li.Tags);
-
         Section = li.Section ?? string.Empty;
-
+        IsSectionOrSeparator = IsSeparator(li);
         UpdateProperty(nameof(Section));
+        UpdateProperty(nameof(IsSectionOrSeparator));
```

Comment:
Not necessarily a todo, but should we start inlining these?

`UpdateProperty(nameof(Section), nameof(IsSectionOrSeparator));`